.eslintrc.json

```diff
@@ -1,3 +1,28 @@
 {
-  "extends": ["amex", "prettier"]
+  "root": true,
+  "extends": ["amex", "plugin:prettier/recommended", "prettier/react"],
+  "overrides": [
+    {
+      "files": ["**/__tests__/**", "**/__mocks__/**", "*.spec.{js,jsx}", "jest.setup.js"],
+      "extends": ["amex/test", "plugin:prettier/recommended", "prettier/react"],
+      "rules": {
+        "import/no-extraneous-dependencies": 0,
+        "no-restricted-globals": 0,
+        "no-underscore-dangle": 0
```

Comment:
Are we sure we don't want to fix this or disable on a case by case basis so we dont have to write exceptions?

Reply:
Good catch, there was only one case using this exception, removed in 824f96d